src/h3lib/lib/h3Index.c

```diff
@@ -214,6 +214,31 @@ void H3_EXPORT(h3ToChildren)(H3Index h, int childRes, H3Index* children) {
     }
 }
 
+/**
+ * h3ToCenterChild produces the center child index for a given H3 index at
+ * the specified resolution
+ *
+ * @param h H3Index to find center child of
+ * @param childRes The resolution to switch to
+ *
+ * @return H3Index of the center child, or 0 if you actually asked for a parent
+ */
+H3Index H3_EXPORT(h3ToCenterChild)(H3Index parentH, int childRes) {
```

Comment:
Nit: I'd prefer just `parent` or `h` to `parentH`. Similarly, below, I'd just go with `child` or `childIndex` - we don't use `-H` as a suffix anywhere else.